postgres/datadog_checks/postgres/data/conf.yaml.example

```diff
@@ -1,56 +1,66 @@
+## All options defined here are available to all instances.
+#
 init_config:
 
+    ## @param service - string - optional
+    ## Attach the tag `service:<SERVICE>` to every metric, event, and service check emitted by this integration.
+    ##
+    ## Additionally, this sets the default `service` for every log source.
+    #
+    # service: <SERVICE>
+
+## Every instance is scheduled independent of the others.
+#
 instances:
 
     ## @param host - string - required
     ## The hostname to connect to.
-    ## NOTE: Even if the server name is "localhost", the agent connects to PostgreSQL using TCP/IP, unless you also
+    ## NOTE: Even if the server name is `localhost`, the Agent connects to PostgreSQL using TCP/IP unless you also
     ## provide a value for the sock key.
     ## If `use_psycopg2` is enabled, use the directory containing the UNIX socket (ex: `/run/postgresql/`)
     ## otherwise, use the full path to the socket file (ex: `/run/postgresql/.s.PGSQL.5433`).
     #
   - host: localhost
 
-    ## @param port - integer - required
-    ## Port to use when connecting to PostgreSQL.
+    ## @param port - integer - optional - default: 5432
+    ## The port to use when connecting to PostgreSQL.
     #
-    port: 5432
+    # port: 5432
 
-    ## @param user - string - required
-    ## Datadog Username created to connect to PostgreSQL.
+    ## @param username - string - required
+    ## The Datadog username created to connect to PostgreSQL.
     #
     username: datadog
 
-    ## @param pass - string - required
-    ## Password associated with the Datadog user.
+    ## @param password - string - optional
+    ## The password associated with the Datadog user.
     #
-    password: <UNIQUEPASSWORD>
+    # password: <PASSWORD>
```

Comment:
Should be `required` ?

Reply:
No:
https://github.com/DataDog/integrations-core/blob/master/postgres/datadog_checks/postgres/postgres.py#L651
```
password = self.instance.get('password', '')
```
https://github.com/DataDog/integrations-core/blob/master/postgres/datadog_checks/postgres/postgres.py#L488
```
            if host == 'localhost' and password == '':
                # Use ident method
                connection_string = "user=%s dbname=%s, application_name=%s" % (user, dbname, "datadog-agent")
```